docs/src/main/paradox/docs/getting-started/notebooks/forge.yml

```diff
@@ -20,7 +20,7 @@ Store:
     project_property: "https://bluebrain.github.io/nexus/vocabulary/project"
   max_connection: 5
   versioned_id_template: "{x.id}?rev={x._store_metadata._rev}"
-  file_resource_mapping: https://raw.githubusercontent.com/BlueBrain/nexus-forge/master/examples/configurations/nexus-store/file-to-resource-mapping.hjson
+  file_resource_mapping: https://raw.githubusercontent.com/BlueBrain/nexus/8851236f5bc38e113f1f04af2ca0bb31dab1c576/docs/src/main/paradox/docs/getting-started/notebooks/file-to-resource-mapping.hjson
```

Comment:
Why is it not https://raw.githubusercontent.com/BlueBrain/nexus/master/docs/src/main/paradox/docs/getting-started/notebooks/file-to-resource-mapping.hjson ?

Reply:
I just follow the idea of having a specific commit in ref.  I will not revise that in this PR.

Reply:
if someone were to edit it hoping to have the edits reflected in its usage, it wouldn't work though?

Reply:
Notebooks have to be updated as well for now. That would be my next PR.